core/third_party_libs/pue/pue-client.php

```diff
@@ -1373,7 +1373,7 @@ private function requestUpdate()
             //oh let's generate the download_url otherwise it will be old news...
 
             if (! empty($this->download_query)) {
-                $d_install_key = $this->install_key_arr['key'];
+                $d_install_key = isset($this->install_key_arr['key']) ? $this->install_key_arr['key'] : NULL;
```

Comment:
lolz... started to panic when I saw a change in `pue-client.php`
but this is a good change!  thnx  👍🏻